src/routes/todos/+page.svelte

```diff
@@ -40,8 +40,8 @@
 		</ul>
 
 		<form action="?/addTodo" method="POST" use:enhance={addTodo}>
-			{#if form?.missing}
-				<div class="flex flex-row justify-between">
+			{#if !form?.success}
+				<div class="flex flex-col justify-between">
```

Comment:
do you prefer the inputs stacked? if so can you change line 69 to flex-col as well? so the forms are the same layout with and without the error

Reply:
Stacked inputs makes more sense for mobile design. I just changed it for usability during testing.. at least till we finalise styling